building/Windows.md

```diff
@@ -0,0 +1,133 @@
+# Windows
+
+If we want we can build Phoenix-rtos images for all targets and run `ia32-generic-qemu` on Windows with Cygwin and WSL.
+
+## Cygwin
+
+In the beginning, we need to download libUSB and set the QEMU environment.
+
+To do that we need to simply download QEMU from the official website which is targeted to our system. The same
+situation is with libUSB, but we need to follow the instructions provided by the creator.
+
+* [QEMU](https://www.qemu.org/download/#windows)
+* [libUSB](https://github.com/libusb/libusb)
+
+Next up we need to download Cygwin from the official website. It is a collection of Open Source tools that provide
+functionality similar to a Linux distribution on Windows.
+
+* [Cygwin](https://cygwin.com/install.html)
+
+The installation process is in this case also a package manager. After selecting mirror we can easily select what
+packages we want, but in larger groups, this can be annoying that's why we only select two elements (They will be
+installed with dependencies) which are mandatory to use package manager in Cygwin terminal.
+
+Those are `Lynx` and `Wget`. Thanks to them, we will be able to install `apt-cyg` and work with a package similar way
+to Linux.
+
+Cygwin has a problem with translating multiple elements at once. Is suggested to use commands separately

```

Comment:
I would simplify this and create as instruction, such as how to install Cygwin. It might also be worthwhile to use a dropdown list.